pkg/zero/connect/connect.proto

```diff
@@ -55,7 +55,10 @@ message EnvoyMetricsRequest {
 }
 
 // PomeriumMetricsRequest is used to request current pomerium metrics
-message PomeriumMetricsRequest {}
+message PomeriumMetricsRequest {
+  // only include metrics that match the provided labels, and relabel them with the provided labels
```

Comment:
this comment doesn't really make sense here, is it outdated?

Reply:
yes we moved relabeling to the cloud side indeed